compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/InheritedDefaultMethodsOnClassesLowering.kt

```diff
@@ -111,8 +112,13 @@ private class InheritedDefaultMethodsOnClassesLowering(val context: JvmBackendCo
             irFunction.body = irBlockBody {
                 +irReturn(
                     irCall(defaultImplFun.symbol, irFunction.returnType).apply {
+                        interfaceImplementation.parentAsClass.typeParameters.forEachIndexed { index, irTypeParameter ->
+                            //TODO: See Note [Interface Parameters]
```

Comment:
Minor, but could you extract a simple method that would return `context.irBuiltIns.anyNType` and put the comment there instead, so that we have only one TODO instead of three?

Reply:
Yes, sure! Will rebase, as well.

Reply:
Done. Thanks!